measure_extinction/extdata.py

```diff
@@ -215,6 +215,7 @@ class ExtData:
     model : dict of key:value with model fitting results, including
         - waves: np.ndarray with the wavelengths used in the fitting
         - exts: np.ndarray with the fitted powerlaw model to the extinction curve
+        - residuals: np.ndarray with the relative residuals, i.e. (data-fit)/data
```

Comment:
I would have expected the residuals to be data-fit.  I would call these the fractional residuals.  For fractional residuals, I think it will be more robust to have them as (data-fit)/fit.  This avoids dividing by the the noise in the data.

Reply:
Ok, I changed the fractional residuals to (data-fit)/fit.